esp/src/eclwatch/FilePartsWidget.js

```diff
@@ -23,10 +23,14 @@ define([
         hpccCommon,
         template) {
 
-        var formatNum = hpccCommon.format(",");
-
         var nlsHPCC = nlsHPCCMod.default;
 
+        const formatNum = (num) => {
+            const format = hpccCommon.format(",");
```

Comment:
Minor:  
1. Keep formatNum outside of the function (it only need to be created once)
2. Rename this func to "safeFormatNum" (or rename the orig formatNum to _formatNum or similar)?